pkg/version/cobra.go

```diff
@@ -89,27 +85,22 @@ func CobraCommandWithOptions(options CobraOptions) *cobra.Command {
 			switch output {
 			case "":
 				if short {
+					_, _ = fmt.Fprintf(cmd.OutOrStdout(), "client version: %s\n", version.ClientVersion.Version)
```

Comment:
I don't think we should change this. Its output mode is `short` for a reason -- it may even be used by people to parse the version

Reply:
I think this standardizes the output of short, as right below this line we output "client version: %s\n" when the remoteVersion is not nil. I'd opt for a consistent output of client version that doesn't change based upon the remote.

Reply:
I would like client version to be the top line in both cases.

Reply:
This doesn't change how the checks are performed, this changes the output to be consistent. Remote checks are still done as they were previously... just if there is a remote check (which is still done), the output for client is "client version: %s" rather than client version being just "%s" when there is no remote, and "client version: %s" when there is.

Reply:
I retract my comment - thanks for the clarification. this lgtm

Reply:
@dhawton Thanks, makes sense to me as well. I just checked kubectl also prints like `client version:` on the top.